frontend/src/admin/header/Header.js

```diff
@@ -11,23 +12,20 @@ import LogoutButton from './LogoutButton';
 
 const StyledHeader = styled.header`
   background-color: ${props => props.theme.primaryColour};
-  display: flex;
-  align-items: center;
   justify-content: space-between;
   height: 60px;
-  padding: 0 50px;
+  padding: 0 30px;
 `;
 
 const Header = ({ canSelectBranch, selectedBranch }) => (
-  <StyledHeader>
-    <img src="/images/rabble-rouser-logo.png" alt="Rabble Rouser" width="90px" />
-    <span id="header-logo" alt="Logo" />
+  <SpacedLayout container={StyledHeader}>
```

Comment:
I know I was unsure about the component-type-as-a-prop thing previously, but this is really growing on me.